Gemfile.lock

```diff
@@ -37,7 +37,7 @@ GEM
       rexml
     kramdown-parser-gfm (1.1.0)
       kramdown (~> 2.0)
-    liquid (4.0.3)
+    liquid (4.0.4)
```

Comment:
You don't need to commit the Gemfile.lock. Can you please omit this file?

Reply:
Hi, did you generate an API key for Ersi Leaflet Api?

Btw, how do you want us to include API keys